src/boot/main.c

```diff
@@ -298,8 +316,11 @@ void handle_dp_complete(void) {
     sCurrentDisplaySPTask->state = SPTASK_STATE_FINISHED_DP;
     sCurrentDisplaySPTask = NULL;
 }
-extern void crash_screen_init(void);
 
+extern void crash_screen_init(void);
+#ifdef RCVI_HACK
+extern OSViMode VI;
```

Comment:
the extern doesn't need to be ifdefed, externs have no effect on compilation if unused